ducktape/tests/runner.py

```diff
@@ -50,6 +51,8 @@ def __init__(self, min_port, max_port):
 
         self.zmq_context = zmq.Context()
         self.socket = self.zmq_context.socket(zmq.REP)
+        # Set a reasonable linger time to help with unclean shutdown
+        self.socket.setsockopt(zmq.LINGER, self.LINGER_MS)
```

Comment:
Just for my understanding, what are the implications for linger, does it just keep the connection open for LINGER_MS amount of time before raising the zmq.Again?

Reply:
How is this differnet from the `self.socket.RCVTIMEO`